marimo/_tutorials/fileformat.py

```diff
@@ -280,170 +286,107 @@ def _():
 
         Modules imported in a setup cell can be used in "top-level" functions or
         classes. You can add the setup cell from the general menu of the editor under:
-
-        ::lucide:diamond-plus:: Add setup cell
+        ::lucide:diamond-plus:: Add setup cell.
         """
     )
     return
 
 
 @app.cell(hide_code=True)
-def _():
+def _(mo):
     mo.md(
         r"""
-        ### Body cells
-
-        Cells that are not saved as functions or classes are serialized as follows:
-
-        ```python
-        @app.cell(hide_code=True) # <- Cell options are also saved to file
-        def _(increment, a): # <- Signatures are automatically generated and consist of used references
-            message = mo.md(f'''
-                `a` incremented is {increment(a)}
-            ''')
-            return (message,) # <- All used cell definitions are returned
-
-        ```
+        ### Functions and classes
 
-        The default name of a cell is `_`, but you can explicitly give cells a
-        name, as seen below:
+        Notebook files expose functions and classes that depend only on variables defined in the setup cell (or on other such functions or classes). For example, the following cell:
         """
     )
     return
 
 
-@app.cell
-def cell_example():
-    runtime_definition = "Defined in a normal cell, executed in runtime."
-
-    print("This cell is a normal body cell!")
+@app.function
+def roll_die():
+    """
+    A reusable function.
 
-    "Notice how the last line is output"
-    return (runtime_definition,)
+    Notice the indicator in the bottom right of the cell.
+    """
+    return random.randint(1, 7)
 
 
 @app.cell(hide_code=True)
-def _():
+def _(mo):
     mo.md(
         r"""
-        ### Functions and classes
-
-        marimo directly exposes functions and classes that depend only on variables defined in the setup cell (or on other such functions or classes). For example:
+        ... is saved in the notebook file as
 
         ```python
         @app.function
-        def say_hello(name="World"):
-            return f"Hello, {name}"
-        ```
-
-        See the following example in the playground below:
-        """
-    )
-    return
+        def roll_die():
+            \"""
+            A reusable function.
 
-
-@app.function
-def function_example():
-    """Notice how an indicator in the lower right corner shows this cell is serialized differently"""
-    return mo.ui.slider(1, 100)
+            Notice the indicator in the bottom right of the cell.
+            \"""
+            return random.randint(1, 7)
+        ```
 
 
-@app.cell(hide_code=True)
-def _():
-    mo.md(
-        r"""
-        Standalone classes are exposed with the `@app.cell_definition` decorator:
+        Making it importable as
 
         ```python
-        @app.class_definition
-        class MyClass: ...
+        from fileformat import roll_die
```

Comment:
`from my_notebook import roll_die`?

Tutorials start in a fake temp dir, and running this directly will fail